### PR TITLE
added various fields where needed and fixed some params

### PR DESCRIPTION
**Pull request by elimelt**
mostly just added an additional field here and there.

### CODE REVIEW

File: server/server/settings.py [hunk 1]
```diff
@@ -158,6 +158,13 @@
...
 
+devclient = 'http://localhost:5173'
+
+if DJANGO_DEBUG == True:
+    CORS_ALLOWED_ORIGINS.append(devclient)
+    CSRF_TRUSTED_ORIGINS.append(devclient)
+
+
 # PROD ONLY
 CSRF_COOKIE_SECURE = True
```
Comment (ShawnCollinge) [inline server/server/settings.py:169]: Should make this `CSRF_COOKIE_SECURE = !DJANGO_DEBUG` as we don't want secure for local.

File: server/server/settings.py [hunk 2]
```diff
@@ -158,6 +158,13 @@
...
 SESSION_COOKIE_SECURE = True
```
Comment (ShawnCollinge) [inline server/server/settings.py:170]: Ditto with here

File: server/server/settings.py (hunk 1 above)
Comment (ShawnCollinge) [reply server/server/settings.py:169]: Might want to change line 155 and 156 too to 

`CSRF_COOKIE_SAMESITE = 'None' if !DJANGO_DEBUG else 'Lax'`
`SESSION_COOKIE_SAMESITE = 'None' if !DJANGO_DEBUG else 'Lax'

Comment (ShawnCollinge) [reply server/server/settings.py:169]: Or does it work locally without this? `COOKIE_SECURE` means it needs to be https

Comment (elimelt) [reply server/server/settings.py:169]: It was working locally. Not sure why though

Comment (ShawnCollinge) [reply server/server/settings.py:169]: Interesting, might be because it's same same site so it doesn't require https.

**Review by ShawnCollinge — APPROVED** [review]
LGTM

**Pull request merged**